src/main/scala/fastq.scala

```diff
@@ -26,36 +35,116 @@ case object fastq {
     def slice(from: Int, until: Int): Sequence =
       Sequence( sequence.slice(from, until), Quality( quality.value.slice(from, until) ) )
 
+    def take(n: Int): Sequence =
+      Sequence( sequence take n, Quality( quality.value take n ) )
+
+    def takeRight(n: Int): Sequence =
+      Sequence( sequence takeRight n, Quality( quality.value takeRight n ) )
+
     def ++(other: Sequence): Sequence =
       Sequence(sequence ++ other.sequence, Quality( quality.value ++ other.quality.value ))
 
-    def asStringPhred33: String =
-      s"${sequence}\n+\n${quality.toPhred33}"
+    def asStringPhred33: String = Seq(
+      sequence,
+      "+",
+      quality.toPhred33
+    ).mkString("\n")
```

Comment:
I think it's always better to use `.mkString` to join several strings with the same symbol, especially lines.